jsapp/js/account/accountFieldsEditor.component.tsx

```diff
@@ -183,14 +224,19 @@ export default function AccountFieldsEditor(props: AccountFieldsEditorProps) {
             />
           </div>
         )}
-      </div>
 
-      <div className={styles.row}>
+        {/* Insert a blank spacer to start a new row for these */}
+        {!!(count % 2) &&
+          isFieldToBeDisplayed('country') &&
+          isFieldToBeDisplayed('city') &&
+          ++count && <div className={styles.field} />}
+
```

Comment:
You might be able to accomplish the same thing that `count` and this empty `<div/>` are doing by sticking each row of 2 options in a flex container. The code works as-is, but some parts of it are hard to parse - for example, it took me a second to realize why it's `++count` instead of `count++`, and I was kind of surprised it was being incremented in the render.

Reply:
For this one, I'd be fine with some additional comments about the `count` logic and renaming it to something more descriptive (e.g. `fieldCount`.)